### PR TITLE
FOUR-3102: Implement LDAP

### PR DESCRIPTION
**Pull request by danloa**
Fixes [https://processmaker.atlassian.net/browse/FOUR-3102](https://processmaker.atlassian.net/browse/FOUR-3102)

### CODE REVIEW

**Review by GooseCoder — APPROVED** [review]
lgtm

File: resources/js/admin/settings/components/SettingsImport.vue [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<template>
+  <div>
+    <b-modal v-model="shown" :title="title" @hidden="onModalHidden">
+      <div>
+        <p>{{ $t('Please upload a settings file that was previously exported from ProcessMaker 4.') }}</p>
+        <p>{{ $t('All settings that are present on this system will be updated with your imported values.') }}</p>
+        <h5 class="text-center pt-3 pb-2" v-if="imported">
+          <i class="fas fa-check-circle text-success"></i> {{ $t("\{\{ count \}\} \{\{ plural \}\} updated.", {count: this.total, plural: this.plural}) }}
```
Comment (caleeli) [inline resources/js/admin/settings/components/SettingsImport.vue:8]: Please move this translated label to a computed property or a method to improve the code reading.

File: resources/js/admin/settings/components/SettingsImport.vue [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          headers: {
+            'Content-Type': 'multipart/form-data'
+          }
+        }
+      ).then(response => {
+        this.imported = true;
+        this.total = response.data.meta.total;
+      })
+      .catch((error) => {
+        console.log('ERROR', error);
```
Comment (caleeli) [inline resources/js/admin/settings/components/SettingsImport.vue:104]: Consider to use `console.error` instead of console.log

**Review by caleeli — CHANGES_REQUESTED** [review]

File: resources/js/admin/settings/components/SettingsImport.vue (hunk 1 above)
Comment (danloa) [reply resources/js/admin/settings/components/SettingsImport.vue:8]: About, readability, it seems to me that it is more appropriate to place the strings where they pertain, in this case they are part of HTML. PM is placing translations in this way across all the application. For example in notifications.vue:

image

@caleeli  What do you think?

File: resources/js/admin/settings/components/SettingsImport.vue (hunk 2 above)
Comment (danloa) [reply resources/js/admin/settings/components/SettingsImport.vue:104]: fixed

**Review by caleeli — APPROVED** [review]

**Review by boliviacoca — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://develop-processmaker.ci.processmaker.net

**Pull request merged**